### PR TITLE
Translation-ja: Add trusted-committer/ja/05-lowering-the-barriers-to-entry

### PR DESCRIPTION
**Pull request by ystk**
This PR is a part of #367 .

### CODE REVIEW

File: trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* InnerSourceコミュニティでは、潜在的な https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] の数が少ない。
+* コントリビュータは、勤務時間内にコントリビューションしたいと考えるため、時間の制約が大きくなる。
+* InnerSourceでの作業は、コントリビュータの正式な目標管理の一部に必ずなるとは限らないため、InnerSourceの作業に費やす時間が目標達成を損なうように見える場合がある。
+
+そのため、Trusted Committersにとって、 https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] のオンボーディングやコントリビューション作成のプロセスを、できるだけスムーズにすることが重要となります。
+役立つことがいくつかあります。
+
+* 各コードリポジトリに、適切な README.md を用意する。
+適切な README.md には、リポジトリの内容と、その使用目的が説明されています。
+さらに、リポジトリ内のソフトウェアの取得、構築、テスト、および使用方法について、ライセンスに関する情報も含めて詳細な手順を提供する必要があります。
```
Comment (johtani) [inline trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc:14]:
```suggestion
さらに、ライセンスに関する情報も含め、リポジトリ内のソフトウェアの取得、構築、テスト、および使用方法について詳細な手順を提供する必要があります。
```
ライセンスは手順ではないので少し違和感がありました。

File: trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] に期待される事をまとめた CONTRIBTING.md を作成する。
+それは、次のような一般的な質問に回答する必要があります。
+** バグレポートや機能リクエストを送付するには、どのようにすれば良いか。
+** 質問がある時に、誰にどのようにコンタクトすれば良いか。
+** コーディングスタイル、ブランチ作成、コミットメッセージに関する決まりはどのようなものか。
+** コントリビューションに対する「完了」の定義は何か。
+** コントリビューションに適用されるプロセスの段階には何があるか。
+** コントリビューションが受け入れられた後に、コントリビューションしたコードに対するサポートには何が期待されるか。
```
Comment (johtani) [inline trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc:22]:
```suggestion
** コントリビューションが受け入れられた後に、コントリビューションしたコードに対するサポートとして、何を期待しているか。
```

File: trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+** 行動規範やコミュニティ運営に関するガイドラインはどのようなものか。
+
+ソフトウェアに内部ライセンスが添付されている場合には（企業によっては法的エンティティ間でソフトウェアを共有する前提条件）、そのライセンスのコピーと、権利および義務について分かりやすい説明を含めてください。
+
+これら文書化の作業に加えて、オープンソースソフトウェアの開発と同様に、潜在的な https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] によってローカルに開発されるソフトウェアの実行とテストが簡単かつ直ぐに行えるようにすべきで、それにより可能な限り少ない労力でコントリビューションの実装と検証を開始することができます。
```
Comment (johtani) [inline trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc:27]:
```suggestion
これら文書化の作業に加えて、オープンソースソフトウェアの開発と同様に、潜在的な https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] によってローカルに開発されるソフトウェアの実行とテストが簡単かつ直ぐに行えるようにすべきです。
それにより可能な限り少ない労力でコントリビューションの実装と検証を開始することができます。
```
長いので2つの文章にするとわかりやすいかと。

File: trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+コントリビューションの作成には、 _共有リポジトリ_ と _フォークアンドジョイン_ の2つの共通モデルがあります。
+どちらにも利点があり、Trusted Committerとしては、潜在的および現在の https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] のさまざまなニーズに応えるために両方のモデルをサポートする必要があります。
+
```
Comment (johtani) [inline trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc:31]:
```suggestion
```
元の文章は1つの節のため

File: trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+コントリビューターは、コントリビューションプロセスやコミュニティ自体について質問することが多く、これらの質問に回答できる担当者が必要となります。
+したがって、どのようなInnerSourceコミュニティでも、このような質問に回答するコンタクトパーソンが1人以上いることが重要となります。
```
Comment (johtani) [inline trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc:33]:
```suggestion
したがって、どのようなInnerSourceコミュニティでも、このような質問に回答する担当者が1人以上いることが重要となります。
```

File: trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+通常はTrusted Committerのグループの誰かがコンタクトパーソンになりますが、そうでなければ「オンコール」のコミュニティメンバーがいることを確認する必要があります。
```
Comment (johtani) [inline trusted-committer/ja/05-lowering-the-barriers-to-entry-ja.asciidoc:34]:
```suggestion
通常はTrusted Committerのグループの誰かが担当者になりますが、そうでなければ「オンコール」のコミュニティメンバーがいることを確認する必要があります。
```

**Review by johtani — CHANGES_REQUESTED** [review]
Great work, @ystk 
I left some comments. Please check my comments!

**Comment by rrrutledge** [comment]
Very fast feedback.  Great work, both!

**Comment by ystk** [comment]
Thanks for the suggestions, @johtani . I applied all suggested changes.

**Review by johtani — APPROVED** [review]
LGTM!

**Comment by lenucksi** [comment]
Thanks for the contribution and the reviews @ystk and @johtani! :tada:  I'm merging this then.

**Comment by rrrutledge** [comment]
Great!  Opened a [pull request](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/122) to get it on the website.

**Pull request merged**